sycl/doc/extensions/supported/sycl_ext_oneapi_dot_accumulate.asciidoc

```diff
@@ -1,74 +1,99 @@
-:language: c
 
-// This is what is needed for C++, since docbook uses c++
-// and everything else uses cpp.  This doesn't work when
-// source blocks are in table cells, though, so don't use
-// C++ unless it is required.
-//:language: {basebackend@docbook:c++:cpp}
+== Notice
+
+[%hardbreaks]
+Copyright (C) 2020-2023 Intel Corporation.  All rights reserved.
 
-== Introduction
+Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks
+of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by
+permission by Khronos.
 
-IMPORTANT: This specification is a draft.
 
-NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by permission by Khronos.
+== Contact
 
-NOTE: This document is better viewed when rendered as html with asciidoctor.  GitHub does not render image icons.
+To report problems with this extension, please open a new issue at:
 
-== Name Strings
+https://github.com/intel/llvm/issues
 
-`SYCL_ONEAPI_dot_accumulate`
 
-This is a placeholder name.
+== Dependencies
 
-== Notice
+This extension is written against the SYCL 2020 revision 7 specification.  All
+references below to the "core SYCL specification" or to section numbers in the
+SYCL specification refer to that revision.
 
-Copyright (c) 2020 Intel Corporation.  All rights reserved.
 
 == Status
 
-Working Draft
+This extension is implemented and fully supported by {dpcpp}.
 
-This is a preview extension specification, intended to provide early access to a feature for review and community feedback. When the feature matures, this specification may be released as a formal extension.
 
-Because the interfaces defined by this specification are not final and are subject to change they are not intended to be used by shipping software products.
+== Overview
 
-== Version
+This extension adds new SYCL built-in functions that may simplify development
+and provide access specialized hardware instructions when a SYCL kernel needs
+to perform a dot product of two vectors followed by a scalar accumulation.
 
-Built On: {docdate} +
-Revision: 3
 
-== Contact
+== Specification
 
-Ben Ashbaugh, Intel (ben 'dot' ashbaugh 'at' intel 'dot' com)
+=== Feature test macro
 
-== Dependencies
+This extension provides a feature-test macro as described in the core SYCL
+specification.  An implementation supporting this extension must predefine the
+macro `SYCL_EXT_ONEAPI_DOT_ACCUMULATE` to one of the values defined in the table
+below.  Applications can test for the existence of this macro to determine if
+the implementation supports this feature, or applications can test the macro's
+value to determine which of the extension's features the implementation
+supports.
 
-This extension is written against the SYCL 1.2.1 specification, Revision v1.2.1-6.
+[%header,cols="1,5"]
+|===
+|Value
+|Description
 
-== Overview
+|1
+|Initial version of this extension.
```

Comment:
Do we really want to call this version one and the initial version?  IMHO this loses history we might want to preserve.  If we do call this version one would it be better to add it as a new document and move the existing document to a "deprecated" or "removed" directory so we don't lose track of it?

Reply:
This is an API version, not a document version.  Normally, I would bump the API version since this PR adds new APIs.  However, we never implemented any API versioning prior to this PR, and I can't go back in time and add it now.  The best I can do is to start versioning at "1" now.

Regarding the loss of history ... this PR doesn't remove any of the old APIs.  They are still retained as "deprecated", and this extension document still describes them.  (In fact, they are described better now than in the previous document.)